src/category_theory/endomorphism.lean

```diff
@@ -2,12 +2,18 @@
 Copyright (c) 2019 Yury Kudryashov. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yury Kudryashov, Scott Morrison, Simon Hudon
-
-Definition and basic properties of endomorphisms and automorphisms of an object in a category.
 -/
 import category_theory.groupoid
 import data.equiv.mul_add
 
+/-!
+# Endomorphisms
+
+Definition and basic properties of endomorphisms and automorphisms of an object in a category.
```

Comment:
A quick definition would be appreciated!

Reply:
See suggestion below.